### ISSUE TITLE
Desktop main menus are gone!

### ISSUE THREAD

**Issue by heathermrauch** [jgraph/drawio#3659]
I accidentally clicked "Simple View" in the desktop version of the draw.io application and I can't figure out how to restore the regular menu bar. I have been searching through the app and googling this issue for over an hour all I can find is how to restore a document, not functionality that was removed from the program. Even the online FAQ tells me to go to the help menu in the app, which IS NOT THERE!!! Please help with instructions on how to restore this menu. The simple menu is clearly not meant for use by anyone actually trying to accomplish real work so I would also like to disable it if possible.
![image](https://github.com/jgraph/drawio/assets/81587916/9e5d4871-7f03-4f91-a4c0-9f8af4e41a5e)

**Comment by heathermrauch** [comment]
I swear, it never fails. As soon as I finally break down, swallow my pride, and ask for help, I find the answer. It should not be this hard to turn off "Simple View". I had to hover over every icon until I found the file menu. I was then able to select "Configuration", which opened a blank dialog box. From there, I had to click the "Preferences" button at the bottom of the dialog, which opened what appeared to be a JSON file in the dialog window. From there, I had to manually scroll through the pre-configured properties until I found one named "simple_view" and deleted it in the hopes that it is now un-selectable in the UI. 
![image](https://github.com/jgraph/drawio/assets/81587916/8cad9f73-1898-48f8-9e2b-39dcbff7edb4)
